fuse_models/include/fuse_models/unicycle_2d_state_cost_function.h

```diff
@@ -235,6 +240,8 @@ class Unicycle2DStateCostFunction : public ceres::SizedCostFunction<8, 2, 1, 2,
       {
         Eigen::Map<fuse_core::Matrix<double, 8, 2>> jacobian(jacobians[5]);
         jacobian = A_.block<8, 2>(0, 0);
+        const fuse_core::Matrix2d R_yaw_inv = fuse_core::rotationMatrix2D(-parameters[1][0]);
+        jacobian.block<2, 2>(0, 0) = jacobian.block<2, 2>(0, 0) * R_yaw_inv;
```

Comment:
I believe this should also work
```suggestion
        jacobian.block<2, 2>(0, 0) *= R_yaw_inv;
```